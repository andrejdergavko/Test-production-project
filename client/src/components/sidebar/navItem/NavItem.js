import React from "react";

function NavItem({ label, iconURL }) {
  return <div>{label}</div>;
}

export default NavItem;
