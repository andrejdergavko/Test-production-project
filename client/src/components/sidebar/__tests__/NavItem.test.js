import React from "react";
import renderer from "react-test-renderer";

import NavItem from "../navItem";

describe("NavItem component:", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<NavItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
