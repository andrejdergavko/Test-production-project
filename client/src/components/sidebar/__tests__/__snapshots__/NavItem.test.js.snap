// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItem component: renders correctly 1`] = `
<div
  className=""
>
  <div
    className=""
  />
  <div
    className=""
  />
</div>
`;

exports[`renders correctly 1`] = `
<div
  className=""
>
  <div
    className=""
  />
  <div
    className=""
  />
</div>
`;
